app/route.rest

```diff
@@ -1,18 +1,23 @@
+# Get all users
```

Comment:
Any reason you're not using POSTMAN? If this is development purposes or debuggering then you need to ignore this file or remove it

Reply:
It is in my .gitignore but everytime it is pushed too

Reply:
You need to first delete it from remote, I gave you a resource